### DISCUSSION TITLE
Missing FiraCode Font

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#876] [Q&A] [answered]
Hi all,
I looked at the [Design Guidelines for Paste](https://paste.twilio.design/getting-started/design/) but didn't see information about the FiraCode font. I have Fira Mono downloaded and installed. Not sure what I'm missing here.

![image](https://user-images.githubusercontent.com/71658977/96916671-b7e95100-145c-11eb-8cfc-6a32fe065725.png)

**Comment by serifluous** [comment]
Ah yeah, that's a bug on our part. Will get a fix merged in for you in a sec.

**Comment by serifluous** [comment, marked as answer]
Fixed and merged!

**Reply by ang-twilio** [reply]
Yay thank you!!
![image](https://user-images.githubusercontent.com/71658977/96926879-0867ab00-146b-11eb-8c80-2f43fb3e00b8.png)
